src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -215,4 +231,42 @@ public void immutableSources() {
 
     Assertion.assertEquals("AutomatedTest", invocationLabelText);
   }
+
+
+  //TODO: add cases for clicking 'yes' and 'no' buttons
+  @Execute(asUser = User.USER)
+  public void verifyGoToSourceEditorClickOnModalBackground() {
+    InfoboxBuilderPage builderPage = new InfoboxBuilderPage().openNew("Infobox_verify_go_to_source")
+        .clickGoToSourceButton();
+
+    Assertion.assertTrue(builderPage.isGoToSourceDialogPresent());
+
+    builderPage.clickGoToSourceModalBackground();
+
+    Assertion.assertTrue(builderPage.isInfoboxBuilderOpened());
+  }
+
+  @Execute(asUser = User.USER)
+  public void verifyGoToSourceEditorNonEditedInfobox() {
+    InfoboxBuilderPage builderPage = new InfoboxBuilderPage().openExisting(
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "builderPage" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1481)